### ISSUE TITLE
beef up the kabab

### ISSUE THREAD

**Issue by bradleyrichter** [brave/browser-laptop#9305]
We have received comments that the kabob is too small/soft and hard to notice.

This will add some weight to it while retaining it's kababness by updating the SVG file.

preview:
![image](https://user-images.githubusercontent.com/13509546/26892994-e3440f44-4b6e-11e7-92b9-e14a51be57b5.png)

new SVG:
![image](https://user-images.githubusercontent.com/13509546/26892714-0414a162-4b6e-11e7-8efc-fe8563e8e718.png)

[menu_btn.svg.zip](https://github.com/brave/browser-laptop/files/1058852/menu_btn.svg.zip)

margin adjustment:
![image](https://user-images.githubusercontent.com/13509546/26893027-fccd5506-4b6e-11e7-8163-d3c6abed4e9e.png)

**Comment by luixxiul** [comment]
@bradleyrichter how about this? It makes the entire 26x26 box clickable in the same way as it is for newTabButton.

![clipboard01](https://user-images.githubusercontent.com/3362943/26890604-14682a3c-4bed-11e7-946c-d3b1ce99c8fa.jpg)

**Comment by bradleyrichter** [comment]
@luixxiul I didn't see a change in with those numbers but I created a new SVG and added some padding to align it under the lion button on macOS but it should work fine on Windows.

**Comment by luixxiul** [comment]
I see. On Windows the picture on the 1st post it looks for me it is blurred like triple `~` aligned vertically. Maybe that is my eye problem.

![clipboard01](https://user-images.githubusercontent.com/3362943/26893436-4b7f2f76-4bf6-11e7-84c3-9a8f1d01f9f6.jpg)

With the SVG file that should not happen though.

**Comment by cezaraugusto** [comment]
@luixxiul I think you got the point, mind taking this one? 

I would make this the same width as bravery icon and apply flex to _middlefy_ it so both can be forever aligned vertically. Also adding extra space will improve a11y. If turns out extra space makes clicking area too big, we could replace extra-padding with extra-margin. SVG will take care of the bad looking image issue. 

Pls ping me if you can't make for some reason and I'll jump in.

**Comment by luixxiul** [comment]
@cezaraugusto OK please, thanks.

**Comment by Jacalz** [comment]
I noticed this on Master and I really don´t like how it looks :-1: 
The thing is that it should be wider, it just looks like the old menu was stangely stretched :sweat_smile:

**Comment by NumDeP** [comment]
Hi @cezaraugusto I have to agree with @Jacalz, I understand this is now a closed issue but it just looks so similar to Chrome's and other types of browsers that try to indicate that, this is hiding something behind it; can't you guys mimic it to look like a lions claw/s in forward slash angle  "/" (in case anybody didn't know what a slash looks like)

I suppose if you also had it slanted, you could maximize the usage of the tab area and use up the top-right edge of the bar, possibly enough to include another tab without an extra tab set for some users depending on their monitor, essentially saving half a millimeter if you go right to the edge.

Furthermore, there's only 7 issues left to close before the next major release of 0.19.x and I think if you did consider it you'd push into backlogs so I think if you do contemplate this design it's best to do so now than later.

(No point for the paste if it's not considered) Could someone paste an image of what of the following keys would look like slanted from the top obviously , ' ` (first one is a comma which I think looks more like a lions claw but at a different angle, second one's an apostrophe and the third one is a key left of "1" on a ThinkPad)
